webdav/Dockerfile

```diff
@@ -0,0 +1,63 @@
+FROM httpd:bullseye
```

Comment:
since this is mostly for dev, maybe we want to point to :latest? so we notice early if something breaks?

Reply:
on it